etl/src/main/java/org/pih/hivmigration/etl/sql/Setup.groovy

```diff
@@ -21,6 +21,7 @@ class Setup extends SqlMigrator {
                 id INT PRIMARY KEY AUTO_INCREMENT,
                 patient_id INT,
                 encounter_id INT,
+                encounter_date DATE,
```

Comment:
Encounter date was very helpful in my testing when I went to look up these issues in the existing HIV EMR.